### ISSUE TITLE
[Support]: Is tmpfs / mem drive running? 

### ISSUE THREAD

**Issue by kdjordjev** [blakeblackshear/frigate#2627]
### Describe the problem you are having

Hello
I was able to install Frigate On Proxmox --> LXC --> Docker (Portainer) with Coral and HW acceleration 
Finally everything seems to be running in this setup

However i wanted to use the "tmpfs" option to decrease the SSD drive wear.
Per my understanding, the Frigate should be writing the  5 min temporary video recordings into the memory, and only save the "clips/motion" events  into the SSD. 

I have the tmpfs enabled as per the Docer Compose File below

However:
- i do not see any message in the frigate output that it has been enabled
- I check the /tmp/cache directory and it does not exist 
- I check my storage volume and i can see the temporary video clips are being recorded on the SSD with the events videos 

Can yo help with how to enable this? I want all temporary video clips to be saved into the RAM and only "clips/events" into the SSD.

K

### Version

(0.9.4-26ae608)

### Frigate config file

```yaml
version: "3.9"
services:
  frigate:
    container_name: frigate
    privileged: true 
    restart: unless-stopped
    image: blakeblackshear/frigate:0.9.4-amd64
    shm_size: "128mb" 
    devices:
      - /dev/bus/usb:/dev/bus/usb 
      - /dev/dri/renderD128 
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /var/lib/docker/volumes/frigate/config:/config:ro
      - /var/lib/docker/volumes/frigate:/media/frigate
      - type: tmpfs 
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    ports:
      - "5000:5000"
      - "1935:1935" # RTMP feeds
    environment:
      FRIGATE_RTSP_PASSWORD: "@@@@@"
      LIBVA_DRIVER_NAME: "radeonsi"
```

### Relevant log output

```shell
[2022-01-11 11:56:09] frigate.app                    INFO    : Starting Frigate (0.9.4-26ae608)
Starting migrations
[2022-01-11 11:56:09] peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
[2022-01-11 11:56:09] peewee_migrate                 INFO    : There is nothing to migrate
[2022-01-11 11:56:09] frigate.mqtt                   INFO    : MQTT connected
[2022-01-11 11:56:09] frigate.app                    INFO    : Output process started: 219
[2022-01-11 11:56:09] frigate.app                    INFO    : Camera processor started for garage: 224
[2022-01-11 11:56:09] frigate.app                    INFO    : Camera processor started for front_of_house: 227
[2022-01-11 11:56:09] ws4py                          INFO    : Using epoll
[2022-01-11 11:56:09] frigate.app                    INFO    : Camera processor started for front_yard: 229
[2022-01-11 11:56:09] frigate.app                    INFO    : Camera processor started for front_side_yard: 230
[2022-01-11 11:56:09] frigate.app                    INFO    : Camera processor started for back_yard: 232
[2022-01-11 11:56:09] frigate.app                    INFO    : Camera processor started for front_door: 233
[2022-01-11 11:56:09] frigate.app                    INFO    : Capture process started for garage: 234
[2022-01-11 11:56:09] frigate.app                    INFO    : Capture process started for front_of_house: 238
[2022-01-11 11:56:09] frigate.app                    INFO    : Capture process started for front_yard: 242
[2022-01-11 11:56:09] frigate.app                    INFO    : Capture process started for front_side_yard: 244
[2022-01-11 11:56:09] frigate.app                    INFO    : Capture process started for back_yard: 247
[2022-01-11 11:56:09] frigate.app                    INFO    : Capture process started for front_door: 253
[2022-01-11 11:56:09] ws4py                          INFO    : Using epoll
[2022-01-11 11:56:09] detector.coral                 INFO    : Starting detection process: 217
[2022-01-11 11:56:09] frigate.edgetpu                INFO    : Attempting to load TPU as usb
[2022-01-11 11:56:12] frigate.edgetpu                INFO    : TPU found
```

### FFprobe output from your camera

```shell
working as expected
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

hikvision

**Comment by kdjordjev** [comment]
Never mind... it is functional. I misunderstood the design

My understanding was it will write all new clips in the tmpfs to reduce the SSD wear, and it would transfer ONLY the clips with events to the SSD.

However it seems it is using the tmpfs only for the current stream and then immediately transfers the mp4 file to the ssd.

**Perhaps this could be a feature request:  Use the tmpfs for storage of all clips and transfer to the SSD ONLY the clips containing the events. (in the case where the user do not require full recordings, only event clips)**

Closing the issue

**Comment by blakeblackshear** [comment]
This is already implemented in the 0.10.6 beta: https://github.com/blakeblackshear/frigate/releases/tag/v0.10.0-beta6

**Comment by kdjordjev** [comment]
Thanks..
I tested it and it works..
Gread job
